compose/ui/ui/src/webMain/kotlin/androidx/compose/ui/viewinterop/InteropView.web.kt

```diff
@@ -48,14 +47,57 @@ actual typealias InteropView = Any
  */
 @ExperimentalComposeUiApi
 @Composable
+fun <T : HTMLElement> HtmlElementView(
+    factory: () -> T,
+    modifier: Modifier = Modifier,
+    update: (T) -> Unit = NoOp,
+    onRelease: (T) -> Unit = NoOp,
+    onReset: ((T) -> Unit)? = null,
+) {
+    InternalHtmlElementView(
+        factory = factory,
+        modifier = modifier,
+        update = update,
+        onRelease = onRelease,
+        onReset = onReset,
+    )
+}
+/**
+ * Compose an [HTMLElement] of class [T] into the UI hierarchy.
+ *
+ * In the current implementation, the HTML element will overlay the canvas area according to the given size
+ * (specified by [modifier]). The HTML element will intercept the input events in that area,
+ * and Compose will not see those events.
+ *
+ * @param factory The block creating the [T] to be composed.
+ *
+ * NOTE: [T] shouldn't be leaked outside and will be managed by Compose runtime efficiently.
+ * Remembering [T] externally and passing it to be returned from [factory] can (and probably will)
+ * lead to hilarious bugs in case [onReset] is not `null`.
+ * @param modifier The modifier to be applied to the layout.
+ * @param update A callback to be invoked every time the state it reads changes.
+ * Invoked once initially and then every time the state it reads changes.
+ * @param onRelease A callback invoked as a signal that the [T] has exited the
+ * composition forever. Use it to release resources and stop jobs associated with [T].
+ * @param onReset If not null, this callback is invoked when this composable node is
+ * reused in the composition instead of being recreated. Use it to reset the state of [T] to
+ * some blank state. This is a function that will be executed instead of [factory] if the node
+ * containing [T] was reused. If null, [T] will not be reused, a new instance of [T] will be created
+ * using [factory] every time this function enters the composition.
+ */
+@Deprecated(
+    "Use HtmlElementView instead",
+    ReplaceWith("HtmlElementView(factory, modifier, update, onRelease, onReset)")
+)
+@Composable
```

Comment:
```suggestion
@ExperimentalComposeUiApi
@Composable
```

After applying this, there will be no need in new api dump - please drop the commit for api dump.

Reply:
done